evm_loader/program/src/state_account.rs

```diff
@@ -164,7 +164,7 @@ impl<'a> State<'a> {
             .map(|c| c.split_at(2))
             .map(|(meta, key)|
                 BlockedAccountMeta {
-                    key: Pubkey::new(key),
+                    key: Pubkey::try_from(key).expect("key is 32 bytes"),
```

Comment:
Same as above. Consider this: "`key` is expected to be 32 bytes long public key".

Reply:
No one will ever see this. This `expect` will never trigger.